.dockerignore

```
@@ -0,0 +1,13 @@
+.github
+.pytest_cache
+.venv
+.vscode
+ansible
+venv
+.env
+.env.template
+.gitignore
+.gitpod.yml
+Dockerfile
+img.png
+README.md
```

One final suggestion:
```suggestion
README.md
**/__pycache__
```
Not vital, and mostly added in case you're not aware that `.dockerignore` has slightly different behaviour to `.gitignore`, in particular in that it _doesn't_ ignore subfolders by default - we need to explicitly reference them with the `**` prefix to mean arbitrary levels of nesting should be considered for this rule. I think that your existing entries wouldn't matters here, but this is one of those issues that can easily lead to weird confusion when some behaviour starts caching oddly!